ansible/roles/util-cfg-service/templates/apps.properties.j2

```diff
@@ -27,6 +27,7 @@ apps.workspace.default-app-categories       = {{ apps.user_subs }}
 apps.workspace.dev-app-category-index       = 0
 apps.workspace.favorites-app-category-index = 1
 apps.workspace.public-id                    = 00000000-0000-0000-0000-000000000000
+apps.workspace.metadata.category.attrs      = rdf:type, http://edamontology.org/has_topic
```

Comment:
That sounds like a good idea to me, but we should be able to add that later without too much trouble.